policy/modules/apps/pulseaudio.te

```diff
@@ -235,7 +235,7 @@ optional_policy(`
 optional_policy(`
 	udev_read_pid_files(pulseaudio_t)
 	udev_read_state(pulseaudio_t)
-	udev_read_db(pulseaudio_t)
+	udev_read_pid_files(pulseaudio_t)
```

Comment:
Already has the permission.

This prompts a broader question: If the domain doesn't already have the /run access, maybe there no longer is a need? Perhaps we should be dropping the permission instead.